src/node/Launcher.ts

```diff
@@ -190,6 +190,7 @@ class ChromeLauncher implements ProductLauncher {
       '--enable-automation',
       '--password-store=basic',
       '--use-mock-keychain',
+      '--enable-blink-features=IdleDetection',
```

Comment:
Let's add a TODO here reminding us to remove it once the feature ships by default.

Reply:
done